staging/src/k8s.io/apiserver/pkg/endpoints/metrics/metrics.go

```diff
@@ -176,6 +176,14 @@ var (
 		},
 		[]string{"request_kind"},
 	)
+	// GOAWAYResponse is a number of requests which server response with GOAWAY frame
+	GOAWAYResponse = compbasemetrics.NewCounter(
+		&compbasemetrics.CounterOpts{
+			Name:           "apiserver_goaway_response",
+			Help:           "Number of requests which server response with GOAWAY frame",
```

Comment:
response -> responded

Reply:
Also what time period? if you see the previous 2, they say `in last second`

Reply:
Looks like we are adding something new `apiserver_goaway_response` so this is not just adding test cases. am i reading this right?

Reply:
Sorry, I used a wrong metrics type. I will fixed it later.

so this is not just adding test cases. am i reading this right? - yes, 2 commits.

Reply:
> Also what time period? if you see the previous 2, they say in last second

The last two metrics are gauges, this is a counter, counters don't have time frames.

However I do think it'd be good to add this metric in a separate PR. I'd like to take the tests first.

The metric is about *connections* (right?), but the other metrics here are about requests. So it's confusing and I think we should talk about that more. Looks like the tests don't depend on this metric, so hopefully it won't be much trouble to send it in a separate PR.